server/queries/migrations/create_score.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS Score (
+    id                  SERIAL PRIMARY KEY,
+    id_game             INT       NOT NULL,
+    id_user             INT       NOT NULL,
+    spotify_track_id    VARCHAR(50) UNIQUE,
+    timestamp           BIGINT    NOT NULL,
+    is_correct          BOOLEAN   NOT NULL,
+    reaction_time       INT       NOT NULL,
```

Comment:
Maybe it's better if we rename reaction_time with the unit we use, like `reaction_time_ms`?